boards/shields/apollo4p_evb_disp_shield_rev2/apollo4p_evb_disp_shield_rev2.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Copyright (c) 2023 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		accel0 = &adxl362;
```

Comment:
```suggestion
		accel0 = &adxl362adxl362_<shield_name>;
```

Reply:
Why the dual "adxl362adxl362". Is that needed for proper reference?
If so, that may explain an error I am seeing elsewhere.

Is there somewhere in the docs that explains ow these devices are referenced?

Reply:
Done